src/Form/WebHookType.php

```diff
@@ -3,24 +3,33 @@
 namespace AppBundle\Form;
 
 use AppBundle\Entity\OAuth\Client;
-use AppBundle\Entity\WebHook\Callback;
+use AppBundle\Entity\WebHook\WebHook;
+use AppBundle\WebHook\Event;
 use Sonata\AdminBundle\Form\Type\CollectionType;
 use Symfony\Bridge\Doctrine\Form\Type\EntityType;
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\Extension\Core\Type\UrlType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class WebHookCallbackType extends AbstractType
+class WebHookType extends AbstractType
 {
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('client', EntityType::class, ['class' => Client::class])
-            ->add('urls', CollectionType::class, [
+            ->add('client', EntityType::class, [
+                'class' => Client::class,
+                'disabled' => true,
+            ])
+            ->add('event', ChoiceType::class, [
+                'placeholder' => '--',
+                'choices' => array_combine(Event::toArray(), Event::toArray()),
```

Comment:
hm i don't see the real gain of `array_combine` and using 2x memory instead of `choice_label` option.

Reply:
It's simpler to write IMO. In this case, it's not ideal. I'll change it.